File: 1 ano/CP3 JAVA/src/CP3/Coordenador.java
package CP3;

public class Coordenador extends Pessoa{
	
	private String cursos[];
	
	public Coordenador(){
		
	}
	
	public Coordenador(String nome, String ra, String disciplinas[]){
		super(nome, ra);
		this.cursos = disciplinas;
	}

	public String[] getCursos() {
		return cursos;
	}

	public void setCursos(String[] disciplinas) {
		this.cursos = disciplinas;
	}
}

File: C#/CP1/CP1-ClassLibrary/Models/Coordenador.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using CP1_ClassLibrary;


namespace CP1_ClassLibrary.Models
{
    public class Coordenador : Funcionario
    {
        private String[] cursos;

        public Coordenador(
            string[] cursos,
            string tempoContrato,
            string cargo,
            string nome,
            string cpf
        ) : base(nome, cpf, tempoContrato, cargo)
        {
            this.cursos = cursos;
        }

        public String[] GetCursos()
        {
            return this.cursos;
        }


        public void SetCursos(String[] cursos)
        {
            this.cursos = cursos;
        }

    }
}
